src/couch/src/couch_httpd_auth.erl

```diff
@@ -253,6 +246,68 @@ jwt_authentication_handler(Req) ->
             Req
     end.
 
+get_roles_claim(Claims) ->
+    Roles_Claim_Name_Param = config:get(
```

Comment:
these names do not follow any convention in the rest of the codebase.

Reply:
Is `RolesClaimNameParam` a better alternative, or what are the conventions?